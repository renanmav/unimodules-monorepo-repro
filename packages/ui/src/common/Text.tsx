import React from 'react';
import {Text as NativeText, TextProps} from 'react-native';

const Text: React.FC<TextProps> = ({children, ...props}) => {
  return <NativeText {...props}>{children}</NativeText>;
};

export default Text;
