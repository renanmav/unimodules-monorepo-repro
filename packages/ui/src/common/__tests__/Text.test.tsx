import React from 'react';
import {render} from '@testing-library/react-native';

import Text from '../Text';

it('should render correctly', async () => {
  const {toJSON} = render(<Text testID="text">Some text</Text>);

  expect(toJSON()).toMatchSnapshot();
});
