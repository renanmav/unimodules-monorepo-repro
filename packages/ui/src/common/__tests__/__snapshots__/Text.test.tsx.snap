// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="css-text-901oao"
  data-testid="text"
  dir="auto"
  onClick={[Function]}
>
  Some text
</div>
`;
